xbmc/settings/MediaSettings.cpp

```diff
@@ -94,7 +94,7 @@ bool CMediaSettings::Load(const TiXmlNode *settings)
       scalingMethod = (int)VS_SCALINGMETHOD_LINEAR;
     m_defaultVideoSettings.m_ScalingMethod = (ESCALINGMETHOD)scalingMethod;
 
-    XMLUtils::GetInt(pElement, "viewmode", m_defaultVideoSettings.m_ViewMode, ViewModeNormal, ViewModeCustom);
+    XMLUtils::GetInt(pElement, "viewmode", m_defaultVideoSettings.m_ViewMode, ViewModeNormal, ViewModeZoom110Width);
```

Comment:
I examined the code further and think I have figured out what is going on. It looks like the last view mode was accidentally excluded from the view mode spinner. I have added a new commit to try to better abstract the view mode order and text shown in the GUI from the constants that are stored in the settings database. This code can easily be tweaked, if necessary, to hide the non-linear mode.